src/critnib/critnib.h

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
```

Comment:
currently we do not use critnib - please remove it (+change makefiles)

Reply:
Done